include/drv_types.h

```diff
@@ -1171,5 +1171,10 @@ __inline static u8 *myid(struct eeprom_priv *peepriv)
 #include <pci_hal.h>
 #endif
 
+static inline bool is_compat_task(void)
```

Comment:
On kernel 4.4 I get a build error complaining about this function being redefined. Looks like kernel 4.10 [has a header with a macro that defines this function[(http://elixir.free-electrons.com/linux/v4.10/source/include/linux/compat.h#L731). Would you mind changing the patch to remove this function, and add an `#include <linux/compat.h>` instead?